docs/html/operators.html

```diff
@@ -102,14 +102,14 @@ <h3> Assignment Operators </h3>
 				<tr>
 					<td>Operator</td><td>Description</td>
 				</tr>
-				<tr> <td>= </td><td> - </td> </tr>
-				<tr> <td>+=</td><td> - </td> </tr>
-				<tr> <td>-=</td><td> - </td> </tr>
-				<tr> <td>*=</td><td> - </td> </tr>
-				<tr> <td>/=</td><td> - </td> </tr>
-				<tr> <td>%=</td><td> - </td> </tr>
-				<tr> <td>++</td><td> - </td> </tr>
-				<tr> <td>--</td><td> - </td> </tr>
+				<tr> <td>= </td><td> assign a value to a variable </td> </tr>
+				<tr> <td>+=</td><td> increment a variable's value </td> </tr>
+				<tr> <td>-=</td><td> decrement a variable's value </td> </tr>
+				<tr> <td>*=</td><td> multiply a variable's value </td> </tr>
+				<tr> <td>/=</td><td> divide a variable's value </td> </tr>
+				<tr> <td>%=</td><td> modulo a variable's value </td> </tr>
```

Comment:
I suppose the explanation for modulo could be better.

Enhancement?

Reply:
I'm unsure if ```+=``` should be described as 'increment ...' as ```x += -1``` could also be written, decrementing the value.